### PR TITLE
Implement add/remove feedback request methods

### PR DESCRIPTION
**Pull request by camerondurham**
### Description

This PR implements the basic functionality of adding feedback to posts and followup posts. The API endpoint `https://piazza.com/logic/api?method=content.add_feedback` and `https://piazza.com/logic/api?method=content.remove_feedback` accept the "type" and "cid" as parameters to check that a post is a "good note", "good question", or "helpful". This PR adds the endpoint to the PiazzaRPC class and the Network class.

I have read the `CONTRIBUTE.md` document but of course did not want to assume you want to include this feature. This may be too trivial to include in the API but it was easy enough for me to implement.

### Usage

This simple program demonstrates how to use the `add_feedback()` and `remove_feedback()` functions in the Network class. 

```python
from piazza_api import Piazza
from time import sleep
p = Piazza()
p.user_login(email="user@uni.edu", password="XXXXXXXXXXX")
csci = p.network("ClassPiazzaID")

# use post ID, the nr field from the API response
csci.add_feedback(450)

# use followup post cid
csci.add_feedback({"id":"feedbackCID"})

sleep(60)

csci.remove_feedback(450)
csci.remove_feedback({"id":"feedbackCID"})
```

### Testing

I tested the functionality by using these functions in the snippet above to verify I could mark/undo a post as a "good question" and "good comment". Please let me know if you require more testing for this feature. 

**Mark as a good question using add_feedback**
Screenshot 2021-03-26 025345_LI

**Mark as a good comment using add_feedback**
Screenshot 2021-03-26 200500

**Undo mark as a good question using remove_feedback**
Screenshot 2021-03-26 025320

**Undo mark as a good comment using remove_feedback**
Screenshot 2021-03-26 201839

### CODE REVIEW

File: piazza_api/rpc.py [hunk 1]
```diff
@@ -226,6 +226,31 @@ def content_delete(self, params):
...
+            data=params
+        )
+        return self._handle_error(r, "Could not create object {}.".format(
+            repr(params)))
+
+    def content_remove_feedback(self, params):
+        """Unmark a post as a good note
+
+        :type params: dict
+        :param params: the paremters to be passed in
```
Comment (hfaran) [inline piazza_api/rpc.py]: parameters*
Fix in other docstrings as well

File: piazza_api/network.py [hunk 2]
```diff
@@ -358,6 +358,54 @@ def delete_post(self, post):
...
+
+        params = {
+            "cid": cid,
+            "type": "tag_good"
+        }
+
+        return self._rpc.content_add_feedback(params)
+
+    def remove_feedback(self, post):
+        """Marks a post as a good note
```
Comment (hfaran) [inline piazza_api/network.py]: Unmarks*

**Review by hfaran — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a couple nits in the docstrings to fix then this should be good to go.

File: piazza_api/network.py (hunk 2 above)
Comment (camerondurham) [reply piazza_api/network.py]: > 
> 
> Thanks for the contribution! Just a couple nits in the docstrings to fix then this should be good to go.

Ah, thanks for catching those. Definitely was typing/copy-pasting those docstrings too quickly. Also, a huge thanks for developing this API. It's been very handy as a TA and easy to use!

**Review by hfaran — APPROVED** [review]
LGTM

**Pull request merged**